app/queries/get_application_choices_for_providers.rb

```diff
@@ -4,9 +4,9 @@ class GetApplicationChoicesForProviders
     :current_course_option,
     :provider,
     :site,
-    application_form: %i[candidate application_references application_work_experiences application_volunteering_experiences application_qualifications application_work_history_breaks],
-    course: %i[provider accredited_provider],
-    course_option: [{ course: %i[provider] }, :site],
+    { application_form: %i[candidate application_references application_work_experiences application_volunteering_experiences application_qualifications application_work_history_breaks] },
```

Comment:
What do the extra brackets do here?

Reply:
I got a Rubocop warning https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/HashAsLastArrayItem so I did what I was told.

Reply:
Ah, can we just have one set of brackets around all three lines then?

Reply:
:+1: https://github.com/DFE-Digital/apply-for-teacher-training/pull/4752/commits/b77fb872384727967d77f65057f9ae7f1e8f63bd